client/displayservers/Wayland/input.c

```diff
@@ -413,3 +413,17 @@ void waylandRealignPointer(void)
   if (!wlWm.warpSupport)
     app_resyncMouseBasic();
 }
+
+void waylandGuestPointerUpdated(double x, double y, int localX, int localY)
+{
+  if (!wlWm.warpSupport || !wlWm.pointerInSurface || app_isCaptureMode())
```

Comment:
Not sure if the `app_isCaptureMode()` check should be done, as we currently can't block the cursor from entering overlapping windows in capture mode.

Reply:
Not sure for Wayland, X11 needs this to prevent window exit when in capture mode due to the trigger of the leave event when warping into a new window.

Reply:
On Wayland, we are currently unable to prevent window exit in capture mode. A future solution using locked pointers should be able to fix this. I am removing the check so that the cursor at least exits predictably for now.